omero/sysadmins/public.txt

```diff
@@ -101,7 +101,7 @@ To set this up on your OMERO.web installation:
 
     ::
 
-       $ bin/omero config set omero.web.public.url_filter '^/(?!webadmin|webclient/(action|logout|annotate_(file|tags|comment|rating|map)|script_ui|ome_tiff|figure_script)|webgateway/(archived_files|download_as))'
+       $ bin/omero config set omero.web.public.url_filter '^/(webadmin/myphoto/|webclient/(?!(action|annotate_(file|tags|comment|rating|map)|script_ui|ome_tiff|figure_script))|webgateway/(?!(archived_files|download_as)))'
```

Comment:
Using https://gist.github.com/will-moore/8315580 as a test, this regex doesn't block ```webclient/logout```. Is that intentional? If you allow users to logout, they will simply get logged in again to /webclient/ right?